invokeai/app/api/routers/board_images.py

```diff
@@ -1,69 +1,126 @@
+            board_id=board_id, image_name=image_name
+        )
         return result
     except Exception as e:
         raise HTTPException(status_code=500, detail="Failed to add to board")
-    
+
+
 @board_images_router.delete(
-    "/",
+    "/{board_id}",
     operation_id="remove_board_image",
     responses={
         201: {"description": "The image was removed from the board successfully"},
     },
     status_code=201,
 )
 async def remove_board_image(
-    board_id: str = Body(description="The id of the board"),
-    image_name: str = Body(description="The name of the image to remove"),
+    board_id: str = Path(
+        description="The id of the board the image is being removed from"
+    ),
+    image_name: str = Body(
+        description="The name of the image to remove from its board"
+    ),
 ):
     """Deletes a board_image"""
     try:
-        result = ApiDependencies.invoker.services.board_images.remove_image_from_board(board_id=board_id, image_name=image_name)
+        result = ApiDependencies.invoker.services.board_images.remove_image_from_board(
+            image_name=image_name
+        )
         return result
     except Exception as e:
         raise HTTPException(status_code=500, detail="Failed to update board")
 
 
-
 @board_images_router.get(
     "/{board_id}",
-    operation_id="list_board_images",
-    response_model=OffsetPaginatedResults[ImageDTO],
+    operation_id="get_all_board_images_for_board",
+    response_model=GetAllBoardImagesForBoardResult,
+)
+async def get_all_board_images_for_board(
+    board_id: str = Path(description="The id of the board"),
+) -> GetAllBoardImagesForBoardResult:
+    """Gets all image names for a board"""
+
+    result = (
+        ApiDependencies.invoker.services.board_images.get_all_board_images_for_board(
+            board_id,
+        )
+    )
+    return result
+
+
+@board_images_router.post(
+    "/{board_id}/add_images",
+    operation_id="create_multiple_board_images",
+    responses={
+        201: {"description": "The images were added to the board successfully"},
+    },
+    status_code=201,
 )
-async def list_board_images(
+async def create_multiple_board_images(
     board_id: str = Path(description="The id of the board"),
-    offset: int = Query(default=0, description="The page offset"),
-    limit: int = Query(default=10, description="The number of boards per page"),
-) -> OffsetPaginatedResults[ImageDTO]:
-    """Gets a list of images for a board"""
+    image_names: list[str] = Body(
+        description="The names of the images to add to the board"
+    ),
+) -> AddManyImagesToBoardResult:
+    """Add many images to a board"""
 
-    results = ApiDependencies.invoker.services.board_images.get_images_for_board(
-        board_id,
+    results = ApiDependencies.invoker.services.board_images.add_many_images_to_board(
+        board_id, image_names
     )
     return results
 
+
+@board_images_router.post(
+    "/{board_id}/remove_images",
```

Comment:
This feels not very RESTful but not sure whats better while still keeping `board_id` in path